components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
```

Comment:
is this a expanding component or and unchecked ratio button?

Reply:
this is unchecking radio buttons, the accordion and expanding sections are built with radio buttons

Reply:
ok so i guess you instanciate an uncheck radio button component and pass it the expander component? then elemenClass does not make sense for me, unless i can use any element here

Reply:
You hit the nail on the head, this can be used for any radio buttons, this class is meant to be reused.

I am using it for 2 components in this checkin

Reply:
but then i guess its not expandingComponent here? it should be someting elese as it could be anything related to radio buttons

Reply:
For now its only for expanding components so it is relevant, if in the future we wanted to use it for other components entirely then yes a mild refactor would be needed, but then we would also have to move the code to a "shared" or "core" folder, which does not exist because we have not reached that point with MSDS yet